app/utils/broadcast.py

```diff
@@ -501,7 +501,7 @@ def redirect_dependent_on_alert_area(broadcast_message):
             message_type="broadcast",
         )
 
-    return redirect(redirect_url)
```

Comment:
This function no longer does what it's called.  I would change it to something like `get_alert_redirect_url` or `calculate_alert_redirect_url`